app/services/status-messages.ts

```diff
@@ -52,12 +52,12 @@ export default class StatusMessages extends Service {
     }
 
     addStatusMessage(message: StatusMessage): void {
-        this.get('nextMessages').push(message);
+        this.nextMessages.push(message);
     }
 
     updateMessages(): void {
         this.setProperties({
-            messages: this.get('nextMessages') || [],
+            messages: this.nextMessages || [],
```

Comment:
maybe a good chance for nullish coalescing:
```suggestion
            messages: this.nextMessages ?? [],
```